.changeset/tidy-flowers-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/react': patch
+---
+
+Filter \_\_forwaredRef from build.
```

Comment:
I think you can remove the `\` in the markdown for the changelog?

Reply:
hmm i dont remember writing the `\` but it's gone now!